reference/debugging.md

```diff
@@ -0,0 +1,229 @@
+# Debugging
+
+If you’re experiencing issues with the Ambassador Edge Stack and cannot diagnose the issue through the "Diagnostics" tab from the [Edge Policy Console](../../about/edge-policy-console), this document covers various approaches and advanced use cases for debugging Ambassador issues.
+
+We assume that you already have a running Ambassador installation. Our examples use the following CRDs as the Ambassador configuration:
+
+?
+
+First, create an example configuration for debugging demonstrations that are available throughout this document.
+
+Then, complete the following debugging options:
+ 
+* Check Ambassador Status
+* Review Ambassador Logs
+* Examine Pod and Container Contents
+
+## Example Configuration
+
+To see a demonstration of how you can debug your Ambassador instance, follow the instructions to create an example mapping configuration.
+
+Note: The following assumes that you deployed Ambassador and the following services from the [quick start installation guide](../../user-guide/install) to a Kubernetes cluster.
+
+**In the command line:**
+
+1. Create a cluster in GKE with RBAC support enabled, and your user account configured correctly. Then run the following:
+
+    ```console
+    $ gcloud container clusters create ambassador-demo --preemptible
+    $ kubectl create clusterrolebinding cluster-admin-binding-new \
+    --clusterrole cluster-admin --user <your_user_name>
+    ```
+
+2. Deploy the latest version of Ambassador following the instructions from [here](../../user-guide/install).
+3. Next, create an Ambassador Service and deploy a basic `httpbin` Ambassador Mapping by saving the following YAML to a file named `ambassador-services.yaml`
+
+   ```yaml
+    ---
+    apiVersion: getambassador.io/v2
+    kind:  Mapping
+    metadata:
+      name:  httpbin
+    spec:
+      prefix: /httpbin/
+      service: httpbin.org
+      host_rewrite: httpbin.org
+    ```
+
+4. Apply this into your cluster with the following command:
+
+    ```console
+    $ kubectl apply -f ambassador-services.yaml
+    ```
+
+You should now be able to utilize this example mapping for debugging demonstrations and purposes.
+
+## Check Ambassador Status
+
+First, check to see if the [Diagnostics](../diagnostics) service is reachable. The following command assumes you have installed Ambassador in the `ambassador` name space: `kubectl port-forward -n ambassador_<pod name> 8877`
+
+If it is successful, try to diagnose your original issue with the Diagnostics Console.
+
+**If it is not successful, complete the following to see if Ambassador is running:**
+
+1. Check the Ambassador deployment with the following: `kubectl get deployments`
```

Comment:
```
kubectl get deployments -n ambassador
```

...and yeah yeah I know, I said `verb [-n] object` before and now I'm listing `verb object [-n]`. Using 

```
kubectl get -n ambassador deployments
```

will work fine, I just think it looks ugly.